services/datamanager/datasync/upload_data_capture_file.go

```diff
@@ -115,3 +160,20 @@ func sendStreamingDCRequests(ctx context.Context, stream v1.DataSyncService_Stre
 
 	return nil
 }
+
+func getFileExtFromImageFormat(res pb.Format) string {
+	switch res {
+	case pb.Format_FORMAT_JPEG:
+		return ".jpeg"
+	case pb.Format_FORMAT_PNG:
+		return ".png"
+	case pb.Format_FORMAT_RAW_DEPTH:
+		return ".dep"
```

Comment:
".depth" to make this match?

Reply:
We may as well make it match what we defined for our custom mime type. `.dep` sounds good, thanks for looking into that.

Reply:
Cool.